features/org.wso2.analytics.is.feature/src/main/siddhi4/README.md

```diff
@@ -1 +1,6 @@
+In order to build the Siddhi4 execution files
```

Comment:
Is this a prerequisite? To build siddhi apps why is the repository need to be cloned?

Reply:
Ack. thank you